src/vue/pages/funds/FundShortDetails.vue

```diff
@@ -0,0 +1,239 @@
+              </td>
+              <td>
+                {{ fund.endTime | formatCalendar }}
+              </td>
+            </tr>
+
+            <tr>
+              <td>
+                {{ 'fund-short-details.soft-cap-title' | globalize }}
+              </td>
+              <td>
+                <!-- eslint-disable-next-line max-len -->
+                {{ { value: fund.softCap, currency: fund.defaultQuoteAsset } | formatMoney }}
+              </td>
+            </tr>
+
+            <tr>
+              <td>
+                {{ 'fund-short-details.hard-cap-title' | globalize }}
+              </td>
+              <td>
+                <!-- eslint-disable-next-line max-len -->
+                {{ { value: fund.hardCap, currency: fund.defaultQuoteAsset } | formatMoney }}
+              </td>
+            </tr>
+
+            <tr>
+              <td>
+                <!-- eslint-disable-next-line max-len -->
+                {{ 'fund-short-details.sell-title' | globalize({ asset: fund.baseAsset }) }}
+              </td>
+              <td>
+                <!-- eslint-disable-next-line max-len -->
+                {{ { value: fund.baseHardCap, currency: fund.baseAsset } | formatMoney }}
+              </td>
+            </tr>
+
+            <tr>
+              <td>
+                {{ 'fund-short-details.video-about-fund-title' | globalize }}
+              </td>
+              <td>
+                <a
+                  v-if="fund.youtubeVideoUrl"
+                  class="fund-short-details__video-btn"
+                  :href="fund.youtubeVideoUrl"
+                  target="_blank"
+                >
+                  {{ 'fund-short-details.view-video-btn' | globalize }}
+                </a>
+                <p v-else>
+                  {{ 'fund-short-details.no-video-msg' | globalize }}
+                </p>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+
+      <button
+        v-ripple
+        class="fund-short-details__view-btn"
+        @click="viewFund"
+      >
+        {{ 'fund-short-details.view-btn' | globalize }}
+      </button>
+    </template>
+
+    <template v-else-if="!isLoadingFailed">
+      <loader :message-id="'fund-short-details.loading-msg'" />
+    </template>
+
+    <template v-else>
+      <p>
+        {{ 'fund-short-details.loading-error-msg' | globalize }}
+      </p>
+    </template>
+  </div>
+</template>
+
+<script>
+import AssetLogo from '@/vue/common/assets/AssetLogo'
+import Loader from '@/vue/common/Loader'
+
+import { Sdk } from '@/sdk'
+
+import config from '@/config'
+
+import { ErrorHandler } from '@/js/helpers/error-handler'
+
+import { SaleRecord } from '@/js/records/entities/sale.record'
+import { AssetRecord } from '@/js/records/entities/asset.record'
+
+export default {
+  name: 'fund-short-details',
+  components: {
+    AssetLogo,
+    Loader,
+  },
+  props: {
+    fund: {
```

Comment:
you have 2 terms describing the same entity: fund and sale. pls use the word fund only for translations

Reply:
Renamed `fund` to `sale`